src/main/java/com/strandls/esmodule/services/impl/ElasticSearchServiceImpl.java

```diff
@@ -255,6 +255,15 @@ public MapQueryResponse update(String index, String type, String documentId, Map
 		return new MapQueryResponse(queryStatus, failureReason);
 	}
 
+	@Override
+	public String esUpdate(String type, String id, String content) throws IOException {
```

Comment:
remove this function, it is not needed. There is  already a generic update function available in the same file. Reuse that, instead of creating something again that is already there.

This function
```
public MapQueryResponse update(String index, String type, String documentId, Map<String, Object> document)
```